packages/jest-runner/corpus.ts

```diff
@@ -18,20 +18,31 @@ import path from "path";
 import fs from "fs";
 
 export class Corpus {
+	static readonly defaultCorpusDirectory = ".cifuzz-corpus";
 	private readonly _inputsDirectory: string;
+	private readonly _corpusDirectory: string;
```

Comment:
Would it be clearer to rename `_corpusDirectory` into something like `_generatedInputsDirectory`? It might be confusing to have `_corpusDirectory` field in a class named `Corpus`

Reply:
We use different names, but basically "done" ;-)